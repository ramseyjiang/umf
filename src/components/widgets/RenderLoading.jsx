import React from "react";
import { Spinner } from "react-bootstrap";

const RenderLoading = () => (
  <div className="text-center">
    <Spinner animation="border" role="status">
      <span className="sr-only">Loading...</span>
    </Spinner>
  </div>
);

export default RenderLoading;