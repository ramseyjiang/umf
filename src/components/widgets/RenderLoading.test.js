import React from "react";
import { mount } from "enzyme";
import RenderLoading from "./RenderLoading";

it("render correctly render loading component", () => {
  const RenderLoadingComponent = mount(<RenderLoading />);
  expect(RenderLoadingComponent).toMatchSnapshot();
  expect(RenderLoadingComponent.find("span").text()).toEqual("Loading...");
});
