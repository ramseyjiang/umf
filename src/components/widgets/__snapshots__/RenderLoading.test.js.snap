// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly render loading component 1`] = `ReactWrapper {}`;
